stock_removal_location_by_priority/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    removal_priority_active = fields.Boolean(
+        default=False,
```

Comment:
That's default

Reply:
What happens with the default?

Reply:
For booleans default is false, you don't need to specify it.